src/main/java/net/mcreator/workspace/Workspace.java

```diff
@@ -294,6 +295,26 @@ public long getMCreatorVersion() {
 		fileManager.close();
 	}
 
+	@Override public boolean equals(Object o) {
+		if (this == o)
+			return true;
+
+		if (o instanceof Workspace other) {
+			try {
+				return Files.isSameFile(getWorkspaceFolder().toPath(), other.getWorkspaceFolder().toPath());
+			} catch (Exception ignored) {

```

Comment:
What kind if exception can happen here?

Reply:
IOE as declared by the method, the same is caught in `FileIO#isFileOnFileList()`.